nav2_msgs/action/ComputePathThroughPoses.action

```diff
@@ -1,3 +1,16 @@
+# Error codes
+# Note: The expected priority order of the errors should match the message order
+int16 NONE=0
+int16 UNKNOWN=1
+int16 TF_ERROR=2
+int16 START_OUTSIDE_MAP=3
+int16 GOAL_OUTSIDE_MAP=4
+int16 START_OCCUPIED=5
+int16 GOAL_OCCUPIED=6
+int16 TIMEOUT=7
+int16 NO_VALID_PATH=8
+int16 NO_WAYPOINTS_GIVEN=9
```

Comment:
I prefer the via point language when talking about points going through states continuously. I think "waypoints" is a loaded term that people usually think of as things to stop at. Given we have the waypoint follower that stops at poses, I think having another name is helpful to separate these ideas